keyboards/yanghu/unicorne/f411/rules.mk

```diff
@@ -1,7 +1,5 @@
 # MCU name
 MCU = STM32F411
 
-# Address of the bootloader in system memory
-# * It is chip dependent, the correct number can be looked up here: (page 359)
-# * http://www.st.com/web/en/resource/technical/document/application_note/CD00167594.pdf
-STM32_BOOTLOADER_ADDRESS = 0x1FFF0000
+# Bootloader selection
```

Comment:
@drashna FYI.

@fauxpark After commited this suggested change, I got the following build error. 

reverting this change would allow me to have a success build.

Shall I revert this change, or does it require sync to latest qmk:master first?

Thanks!

```
Linking: .build/yanghu_unicorne_f411_default.elf                                                    [ERRORS]
 |
 | /usr/lib/gcc/arm-none-eabi/9.2.1/../../../arm-none-eabi/bin/ld: .build/obj_yanghu_unicorne_f411_default/protocol/chibios/main.o: in function `early_hardware_init_pre':
 | /home/huyang/repo/qmk_firmware/tmk_core/protocol/chibios/main.c:118: undefined reference to `enter_bootloader_mode_if_requested'
 | collect2: error: ld returned 1 exit status
 |
```

Reply:
You may need to update your repo to get it to work properly.

Also, gcc 9 is prone to compiler issues.

Reply:
I [rebased this PR onto QMK master](https://github.com/bcat/qmk_firmware/commits/unicorne) and it seems to build just fine. (I dropped the merged commits for a cleaner history, but I guess another `git merge master` would work just as well.) Haven't tested on a physical keyboard yet, but can do so tomorrow.

Reply:
Okay, I've tested the latest version of this PR on my Unicorne. After rebasing against master and apply the `OLED_DRIVER` suggestions I left, everything works just fine on the physical keeb. :)